budcamp/Homework3/Homework3/task2/TestContainer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class TestContainer
+    {
+        private Dictionary<string, List<Test>> dictOfTests;
```

Comment:
Although it makes sense to me, but I don't think that TestContainer should internally contain a dictionary.

I would imagine the class to be like this:
```
internal class TestContainer {
   public string Id { get; private set; }
   public string Topic { get; private set; }
   public List<Test> Tests { get; private set; }

   public TestContainer(string id, string topic) {
     this.Id = id;
     this.Topic = topic;
     this.Tests = new List<>();
   }

   public void AddTest(test) {
        Tests.Add(test);
   }
}
```